step-capstone/src/App.js

```diff
@@ -6,7 +6,7 @@ class App extends React.Component {
   render () {
     return(
       <div className="App">
-        <TravelObject type="flight" />
+        <TravelObject type="event" data={data} />
       </div>
     );
   }
```

Comment:
could you pull this branch down and git checkout this file? Its gonna definitely cause a merge conflict with App and doesnt need to be committed